lib/index.css

```diff
@@ -16,14 +16,149 @@
 
 @import url(lib/bike-rental.css);
 
+@import url(react-sliding-pane/dist/react-sliding-pane.css);
+
+
 /* Hide IE/Edge clear button in text input fields. */
 input[type=text]::-ms-clear {
     display: none;
 }
 
+/* New app menu */
+.app-menu-icon {
+  width: 21px;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  cursor: pointer;
+  position: absolute;
+  z-index: 10;
+  transition: all 1s ease;
+  top: 16px;
+}
+
+.menu-top-line, 
+.menu-middle-line, 
+.menu-bottom-line,
+.menu-left-x,
+.menu-right-x {
+  border-bottom: 3px solid #ffffff;
+  transition: all 0.5s ease;
+}
+
+.menu-left-x {
+  transform: rotate(45deg);
+  top: 7px;
+  position: relative;
+}
+
+.menu-right-x {
+  transform: rotate(-45deg);
+  bottom: 5px;
+  position: relative;
+}
+
+.slide-pane {
+  transition: all 0.2s ease-in-out;
+}
+
+.slide-pane_from_left {
+  margin: 52px auto 0 0;
+}
+
+.slide-pane__header,
+.slide-pane__close {
+  display: none;
+}
+
+.slide-pane__content {
+  padding: 24px 0;
+}
+
+.app-menu {
+  margin: 0;
+  padding: 0;
+}
+
+.app-menu li {
+  padding: 1rem 2rem;
+  cursor: pointer;
+}
+
+.app-menu li img {
+  width: 25px;
+  margin-right: 2rem;
+}
+
+.app-menu li a {
+  color: inherit;
+  text-decoration: none;
+  outline: none;
+}
+
+/* increase icon size */
+.app-menu li a .fa,
+.expand-submenu-button span .fa {
+  font-size: 2rem;
```

Comment:
Personal preference is 1 rem

Reply:
Are you sure?

<img width="320" alt="Screen Shot 2021-09-14 at 3 00 58 PM" src="https://user-images.githubusercontent.com/86619099/133271784-18803eea-7b55-4822-99bc-fc304d08adcb.png">

Reply:
Oops I meant `margin-right` for `.app-menu span:not(.expand-menu-chevron)` that's just below.